builder/Dockerfile

```diff
@@ -1,6 +0,0 @@
-ARG builder_image
-FROM $builder_image
-RUN apk --update --no-cache add ca-certificates
-RUN apk --no-cache add make git
```

Comment:
Is the zig image used already providing these deps? I see that you added `make` explicitly to be added every time the container is invoked to build rather than leveraging an image.

I'm not familiar with the project but `git` / `ca-certificates` may have served a purpose. Some projects will have build scripts in the source / packages that compile that can leverage that if provided with a git repo, which the build container is mounting. Just because a build is successful without that doesn't mean it didn't introduce a regression to the expectation of builds (such as those not tagged/versioned).